instrumentation/github.com/aws/aws-sdk-go/service/config/config.go

```diff
@@ -0,0 +1,75 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package config sets options for the aws sdk instrumentation
+package config
+
+import (
+	"go.opentelemetry.io/otel"
+	otelmetric "go.opentelemetry.io/otel/api/metric"
+	oteltrace "go.opentelemetry.io/otel/api/trace"
+)
+
+// Config provides options for the aws sdk instrumentation.
+type Config struct {
+	TracerProvider           oteltrace.TracerProvider
+	MetricProvider           otelmetric.MeterProvider
+	Propagators              otel.TextMapPropagator
+	SpanCorrelationInMetrics bool
```

Comment:
This uses the term metrics as a noun instead of an adjective, something we try to avoid. Can we rename this?

```suggestion
	SpanCorrelation bool
```